compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/viewinterop/InteropViewFactoryHolder.kt

```diff
@@ -19,7 +19,6 @@ package androidx.compose.ui.viewinterop
 import androidx.compose.runtime.ComposeNodeLifecycleCallback
 import androidx.compose.ui.InternalComposeUiApi
 
-// TODO: Rename to InteropViewHolder: No need for typealiases and it's not a factory
```

Comment:
https://youtrack.jetbrains.com/issue/CMP-7382/Refactor-InteropViewHolder-InteropViewContainer-in-commonMain